arch/arm/src/efm32/efm32_i2c.c

```diff
@@ -513,7 +513,7 @@ static useconds_t efm32_i2c_tousecs(int msgc, struct i2c_msg_s *msgs)
    * factor.
    */
 
-  return (useconds_t) (CONFIG_EFM32_I2C_DYNTIMEO_USECPERBYTE * bytecount);
+  return (useconds_t) (CONFIG_EFM32_I2C_DYNTIMEOUT_USECPERBYTE * bytecount);
```

Comment:
Oah... I didn't expect so radical changes.
I think thinking only about changing
```
#ifdef CONFIG_EFM32_I2C_DYNTIMEOUT
static useconds_t efm32_i2c_tousecs(int msgc, struct i2c_msg_s *msgs);
#endif /* CONFIG_EFM32_I2C_DYNTIMEOUT */
```
to 
```
#ifdef CONFIG_EFM32_I2C_DYNTIMEO
static useconds_t efm32_i2c_tousecs(int msgc, struct i2c_msg_s *msgs);
#endif /* CONFIG_EFM32_I2C_DYNTIMEO */
```

Reply:
I mean I'm fine with `I2C_DYNTIMEO` (and not changing to `I2C_DYNTIMEOUT`), just would like to have same macro used in all places

Reply:
Ok, change CONFIG_EFM32_I2C_DYNTIMEOUT to CONFIG_EFM32_I2C_DYNTIMEO instead.